app/src/main/res/values-fr/strings.xml

```diff
@@ -11,8 +11,320 @@
+  <string name="consciousness">Conscience</string>
+  <string name="condition">État</string>
+  <string name="cancel_sync">Annuler synchronisation</string>
+  <string name="cancelling_sync">Annulation de la synchronisation…</string>
+  <string name="sync_starting">Synchronisation partir…</string>
+  <string name="troubleshoot_wifi_disabled">Wifi est désactivé</string>
+  <string name="troubleshoot_wifi_disabled_action_enable">Activer</string>
+  <string name="troubleshoot_wifi_disconnected">Wifi est déconnecté</string>
+  <string name="troubleshoot_wifi_disconnected_action_connect">Connectez</string>
+  <string name="troubleshoot_server_auth">Serveur nom d\'utilisateur / mot de passe peut être incorrect</string>
+  <string name="troubleshoot_server_auth_action_check">Vérifier</string>
+  <string name="troubleshoot_server_address">Adresse du serveur peut être incorrect</string>
+  <string name="troubleshoot_server_address_action_check">Vérifier</string>
+  <string name="troubleshoot_server_unreachable">Serveur inaccessible</string>
+  <string name="troubleshoot_action_more_info">Plus d\'information</string>
+  <string name="troubleshoot_server_unreachable_details">
+      Le serveur n\'a pas pu être atteint. Cela peut être parce que:
+      \n
+      \n • Le réseau wifi est incorrecte.
+      \n • L\'URL du serveur est incorrecte.
+      \n • Le serveur est en panne.
+      \n
+      \n S\'il vous plaît contacter un administrateur.
+  </string>
+  <string name="troubleshoot_server_unstable">Server peut être instable</string>
+  <string name="troubleshoot_server_unstable_details">
+      Le serveur répond actuellement avec le code d\'erreur 500, indiquant que le serveur peut être dans un état d\'erreur.
+      \n
+      \n S\'il vous plaît contacter un administrateur.</string>
+  <string name="troubleshoot_server_not_responding">Serveur ne répond pas</string>
+  <string name="troubleshoot_server_not_responding_details">
+      Le serveur est actuellement ne répond pas aux demandes. Cela peut être parce que:\n
+      \n • Le serveur est momentanément indisponible.
+      \n • Le serveur est en cours de démarrage.
+      \n
+      \n S\'il vous plaît contacter un administrateur.</string>
+  <string name="patient_location_error_unknown">Une erreur inconnue est survenue lors de la mise à jour se trouve le patient. Veuillez réessayer.</string>
+  <string name="patient_location_error_interrupted">Patient mise à jour de l\'emplacement a été interrompu. Veuillez réessayer.</string>
+  <string name="patient_location_error_network">Erreur de mise à jour du patient; s\'il vous plaît vérifier que vous êtes connecté au serveur.</string>
+  <string name="patient_location_error_no_such_patient">Mise à jour de localisation d\'un patient Erreur: ce patient ne existe pas sur le serveur.</string>
+  <string name="troubleshoot_package_server_unreachable">Serveur de Package inaccessible</string>
```

Comment:
what is unreachable ? the package or the server ? if it`s the package, maybe sould be better 'Package du serveur inaccessible'